packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx

```diff
@@ -321,6 +315,21 @@ export const menuTabs: ForumOptions<Array<MenuTab>> = {
       tooltip: 'Join a group near you or meet others online',
       showOnMobileStandalone: false,
       showOnCompressed: true
+    }, {
+      id: 'local-groups',
+      title: 'Local Groups',
+      link: '/community',
+      subItem: true,
+    }, {
+      id: 'online-groups',
+      title: 'Online Groups',
+      link: '/community#online',
+      subItem: true,
+    }, {
+      id: 'community-members',
+      title: 'Community Members',
+      link: '/community#individuals',
+      subItem: true,
```

Comment:
I figured since I was removing the "Take Action" sidebar item, I could replace it with these? :) I'm mostly curious to see if people would click on these, so if you'd prefer we not list them then I'm happy to remove them in a couple weeks.